test/util/malicious/app.go

```diff
@@ -0,0 +1,67 @@
+package malicious
```

Comment:
nit: I'd prefer to call this package faulty for capturing general faulty behaviour but YMMV